Code/GraphMol/FileParsers/testPropertyLists.cpp

```diff
@@ -7,8 +7,6 @@
 //  of the RDKit source tree.
 //
 
-#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do
```

Comment:
Unfortunately this file ended up with a bunch of formatting changes.
The only substantive change here is the removal of this `#define`